astronomy/trappist_preoptimisation_initial_state_jd_2457000_000000000.proto - Copy.txt

```diff
@@ -0,0 +1,101 @@
+# Most of the data are from  "The nature of the TRAPPIST-1 exoplanets", Grimm et
```

Comment:
I'm not convinced by the extension `.proto - Copy.txt`.

Reply:
Done.